tap_closeio/streams.py

```diff
@@ -165,13 +165,25 @@ def sync_activities(ctx):
     # calls that are in progress _while_ an extraction is happening, no
     # matter the replication frequency or call duration.
     offset_secs = ctx.config.get("activities_window_seconds", (60 * 60 * 24))
+    date_window = int(ctx.config.get("activities_date_window", 15))
     LOGGER.info("Using offset seconds {}".format(offset_secs))
     start_date -= timedelta(seconds=offset_secs)
-    # date_created__gt has precision to the second
-    formatted_start = start_date.strftime("%Y-%m-%dT%H:%M:%S")
-    params = {"date_created__gt": formatted_start}
-    request = create_request(IDS.ACTIVITIES, params=params)
-    paginated_sync(IDS.ACTIVITIES, ctx, request, formatted_start)
+
+    window_start_date = start_date._datetime
+    while window_start_date <= datetime.utcnow().replace(tzinfo=timezone.utc):
```

Comment:
take datetime.utcnow().replace(tzinfo=timezone.utc) in one variable else it will change in next iteration.

Reply:
Added a variable for now date